### PR TITLE
Fix button alignment on login

### PR DESCRIPTION
**Pull request by nico3333fr**
Buttons were not aligned properly, see

image

- added a flex wrapper and magic

image

### CODE REVIEW

File: src/app/components/layout/SignInLayout.js [hunk 1]
```diff
@@ -16,9 +16,11 @@ const SignInLayout = ({ children, title }) => {
             <PublicHeader
                 action={
                     <>
-                        <SupportDropdown className="pv-button-greenborder-dark" />
-                        <Link className="ml1 notablet pm-button--primary" to="/signup">{c('Link')
-                            .t`Sign up for free`}</Link>
+                        <div className="flex flex-justify-end">
+                            <SupportDropdown className="pv-button-greenborder-dark" />
```
Comment (EpokK) [inline src/app/components/layout/SignInLayout.js]: Is `pv-button-greenborder-dark` can be define with a primary class?

Comment (nico3333fr) [reply src/app/components/layout/SignInLayout.js]: Not yet, I update the Design system and app to enable it.

Comment (EpokK) [reply src/app/components/layout/SignInLayout.js]: Alright, once it's done ping me.

Comment (nico3333fr) [reply src/app/components/layout/SignInLayout.js]: @EpokK this is added on Design System, and fixed here: https://github.com/ProtonMail/proton-vpn-settings/pull/123/commits/30ab2e2825a7260d064dd9e4a91be6f9f8acb13a

**Pull request merged**